pinot-core/src/main/java/org/apache/pinot/core/common/BlockDocIdSet.java

```diff
@@ -78,4 +85,10 @@ default BlockDocIdSet toNonScanDocIdSet() {
 
     return this;
   }
+
+  enum CardinalityEstimate {
```

Comment:
Consider changing it to `isAlwaysTrue()` and `isAlwaysFalse()`, which is similar to `PredicateEvaluator`